servers/rendering/rendering_device_binds.h

```diff
@@ -396,8 +404,10 @@ class RDShaderFile : public Resource {
 							"tesselation_evaluation",
 							"compute"
 						};
-
-						ERR_PRINT("Error parsing shader '" + p_file + "', version '" + String(E.key) + "', stage '" + stage_str[i] + "':\n\n" + error);
+						const String error_context = "Error parsing shader '" + p_file + "', version '" + String(E.key) + "', stage '" + stage_str[i] + "'";
+						// Including the error context in both outputs ensures errors can be matched even if multiple errors occur simultaneously.
+						ERR_PRINT(error_context);
+						print_error(error_context + ":\n\n" + error);
```

Comment:
Doesn't that lead to reporting the error twice in stdout / output dialog?

Reply:
Yes, technically, but just the first line. As mentioned in the comment, the error messages can be printed out of order, so I figured for debugging it's probably the most useful if the file/version/stage is printed again right before the shader code dump.